src/main/java/pl/javastart/task/TaxCalculator.java

```diff
@@ -0,0 +1,54 @@
+package pl.javastart.task;
+
+public class TaxCalculator {
+    Product product;
```

Comment:
To nie jest potrzebne, bo przekazujesz produkt jako argument metody getBruttoPrice